RELEASE_NOTES.md

```diff
@@ -7,51 +7,37 @@ Example Note:
 Details about the thing that changed that needs to get included in the Release Notes in markdown.
 -->
 
-# knife-azure 1.7.0 release notes:
-In this release, few options have been removed since their support has been removed from the azure-chef-extension. The options are: `--auto-update-client`, `--uninstall-chef-client` and `--delete-chef-extension-config`. They were earlier supported with `cloud-api` bootstrap protocol which uses azure-chef-extension for installing chef-client.
+# knife-azure 1.8.0 release notes:
+In this release `--chef-service-interval` option is renamed to `--chef-daemon-interval`.
```

Comment:
Perhaps we can mention that we have added `daemon` option and renamed `--chef-service-interval` to `--chef-daemon-interval`.